primo/data_parser/input_options_well_data.py

```diff
@@ -0,0 +1,214 @@
+    ),
+)
+CONFIG.declare(
+    "fill_depth",
+    ConfigValue(
+        default=1000,
+        # Assuming that no well is deeper than 20,000 ft
+        domain=InRange(0, 20000),
+        doc="Value to fill with, if the depth is missing",
+    ),
+)
+
+for key, val in binary_type_metrics.items():
+    CONFIG.declare(
+        "fill_" + key,
+        ConfigValue(
+            default=val,
+            domain=Bool,
+            doc=(
+                f"Value to fill with, if {SUPP_IMPACT_METRICS[key].full_name} "
+                f"information is missing"
+            ),
+        ),
+    )
+
+CONFIG.declare(
+    "fill_hospitals",
+    ConfigValue(
+        default=0,  # Assuming no hospitals nearby
+        domain=NonNegativeInt,
+        doc=("Value to fill with, if number of Hospitals nearby " "is not specified"),
+    ),
+)
+CONFIG.declare(
+    "fill_schools",
+    ConfigValue(
+        default=0,  # Assuming no hospitals nearby
+        domain=NonNegativeInt,
+        doc=("Value to fill with, if number of Schools nearby " "is not specified"),
+    ),
+)
+CONFIG.declare(
+    "fill_state_dac",
+    ConfigValue(
+        default=0.0,
+        domain=NonNegativeFloat,
+        doc=("Value to fill with, if the state DAC information " "is not specified"),
+    ),
+)
+CONFIG.declare(
+    "fill_ann_gas_production",
+    ConfigValue(
+        default=0.0,  # No gas production, if not specified
+        domain=NonNegativeFloat,
+        doc=(
+            "Value to fill with, if the annual gas production [in Mcf/yr] "
+            "is not specified"
+        ),
+    ),
+)
+CONFIG.declare(
+    "fill_ann_oil_production",
+    ConfigValue(
+        default=0.0,  # No oil production, if not specified
+        domain=NonNegativeFloat,
+        doc=(
+            "Value to fill with, if the annual oil production [in bbl/yr] "
+            "is not specified"
+        ),
+    ),
+)
+CONFIG.declare(
+    "fill_five_year_gas_production",
+    ConfigValue(
+        default=0.0,  # No gas production, if not specified
+        domain=NonNegativeFloat,
+        doc=(
+            "Value to fill with, if the five-year gas production [in Mcf] "
+            "is not specified"
+        ),
+    ),
+)
+CONFIG.declare(
+    "fill_five_year_oil_production",
+    ConfigValue(
+        default=0.0,  # No oil production, if not specified
+        domain=NonNegativeFloat,
+        doc=(
+            "Value to fill with, if the five-year oil production [in bbl] "
+            "is not specified"
+        ),
+    ),
+)
+
+# Well data categorization approach
+CONFIG.declare(
+    "well_type_partition",
+    ConfigValue(
+        default=True,
+        domain=Bool,
+        doc="Partition the well data based on well type: oil/gas",
+    ),
+)
+CONFIG.declare(
+    "depth_threshold",
+    ConfigValue(
+        default=2000.0,
+        domain=NonNegativeFloat,
+        doc="Depth value for classifying wells into shallow or deep",
```

Comment:
Minor suggestion to add units. ("Depth value in feet for...")

Reply:
I have removed these options from `ConfigDict`. I'm planning to add them directly to the method where they are needed.  This way, it will be convenient for the users. I will add the units in the doc string of that method. This will be in the next PR.

Reply:
I think keeping this in ConfigDict makes sense. This is a user option we need from the end user.